src/Shell/Task/ExtractTask.php

```diff
@@ -122,7 +122,8 @@ protected function _getPaths()
             $response = $this->in($message, null, $defaultPath);
             if (strtoupper($response) === 'Q') {
                 $this->err('Extract Aborted');
-                return $this->_stop();
+                $this->_stop();
+                return;
```

Comment:
Okay, as for `Inflector::reload()` if it's needed it can be put in bootstrap.php or functions.php file in `Utlility/` since `Utility` is a standalone package.